en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
 > **Warning** This topic is under construction. It is not complete and has not been fully reviewed.
 
-During setup you will be asked to assign a Mode Switch on your Remote Control. With it you will be able to switch flight modes during flight and ask the aircraft to act in a different way depending on your needs.
+Flight modes are used to provide different types of autopilot control, enabling manual, assisted and fully autonomous flight.
 
-Depending on the amount of assistance the Flight controller commits to you during flight you will have three main flight modes categories that you can use in PX4.
+Pilots can transition between flight modes using switches on the remote control or with a ground control station (see [Flight Mode Configuration](../config/flight_mode.md)).
 
-   **• Manual Modes**
+This topic provide a very high level view of each of the flight modes, and the (mostly minor) differences in how they behave in copter, fixed-wing and VTOL frames. For more detailed information see [Flight Modes](../flight_modes/README.md).
 
-   **•	Assisted Modes**
+> **Tip** ![GPS](../../images/flight_modes/GPS_s.png) - This symbol is used to denote modes that *require* position information! These modes cannot be enabled without a position lock from GPS, VIO, or some other positioning system.
 
-   **•	Autonomous Modes**
+## Copter
 
-You have to remember that the flight modes have a slightly different way of behavior depending on the air-frame. There are differences between Airplane, Multicopter and VTOL modes so we will explain it in brief on this page.
+### Manual Modes
 
-Some modes are GPS guided (dependent) and we will use ![GPS](../../images/flight_modes/GPS_s.png) symbol to denote them. In order to function properly and be enabled, this GPS dependent mode’s need a GPS lock and they cannot be enabled during flight without GPS Lock. 
+Manual modes are those where the user has control over vehicle movement via the RC control sticks (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode. For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
 
-> **Warning** It is highly advisable when you start  your aircraft at the flight field, to wait for GPS Lock prior arming to have a good Home Point Location Lock. You could see it when the Pixhawk LED starts blinking green or as a "H" symbol apears in the QGroundControl. Without Home Point GPS Lock it is impossible for the **Return to Launch / Return to Home** to be engaged and the craft will never come back in case of emergency.
-You can read in depth about Flight Modes in the Flight Mode Setup page.
+#### Manual/Stabilized Mode
```

Comment:
FYI on Copter you can select either "Manual" or "Stabilised" modes in QGC. They are the same thing.

Reply:
I know they are the same, But this is a concept hard to grasp for the novice. IE. When we say Manual for MC the novice usually understands it is Manual as in airplane. So that is why I left the Stabalized. Long time ago when we changed the names of the modes by our request instead of PosCtl - Position etc, I asked if we can make consistency between the airplane mode names and copter mode name. But for the MC they decided to leave it as is, because Manual has a different meaning regarding the FMU IO operation. Anyway

Reply:
Thanks for clarification. Then perhaps we call this "Manual" mode, but add a "Tip" stating that this same mode will be selected if you choose "Stabilized" in QGC (?)

Reply:
I think Manual/Stabalized is good enough. This way they will associate the two terms without tips etc.